go-controller/pkg/ovn/master.go

```diff
@@ -599,10 +599,10 @@ func (oc *Controller) addNodeAnnotations(node *kapi.Node, subnet *net.IPNet) err
 	return nil
 }
 
-func (oc *Controller) addNode(node *kapi.Node) (hostsubnet *net.IPNet, err error) {
+func (oc *Controller) addNode(node *kapi.Node) (*net.IPNet, error) {
 	oc.clearInitialNodeNetworkUnavailableCondition(node, nil)
 
-	hostsubnet, _ = util.ParseNodeHostSubnetAnnotation(node)
+	hostsubnet, _ := util.ParseNodeHostSubnetAnnotation(node)
```

Comment:
should we be logging the error returned?

Reply:
@trozet no, because the function returns an error if the node has no hostsubnet yet so it would be useless logging for every node the first time. Other places though, yes.